config/config.proto

```diff
@@ -34,6 +34,17 @@ message PushoverConfig {
   optional string user_key = 2;
 }
 
+message HipChatConfig {
+  // Hipchat auth token, https://www.hipchat.com/docs/api/auth
+  optional string auth_token = 1;
+  // Hipchat room id, https://www.hipchat.com/rooms/ids
+  optional int32 room_id = 2;
+  // color of message
+  optional string color = 3 [default = "red"];
+  // should this message notify or not
```

Comment:
in hipchat you can send a message to a room, and it will not make a notification for others